frontend/models/PasswordResetRequestForm.php

```diff
@@ -41,20 +41,30 @@ public function sendEmail()
             'email' => $this->email,
         ]);
 
-        if ($user) {
-            if (!User::isPasswordResetTokenValid($user->password_reset_token)) {
-                $user->generatePasswordResetToken();
-            }
-
-            if ($user->save()) {
-                return \Yii::$app->mailer->compose(['html' => 'passwordResetToken-html', 'text' => 'passwordResetToken-text'], ['user' => $user])
-                    ->setFrom([\Yii::$app->params['supportEmail'] => \Yii::$app->name . ' robot'])
-                    ->setTo($this->email)
-                    ->setSubject('Password reset for ' . \Yii::$app->name)
-                    ->send();
-            }
+        if (!$user) {
+            return false;
+        }
+        
+        if (!User::isPasswordResetTokenValid($user->password_reset_token)) {
+            $user->generatePasswordResetToken();
+        }
+        
+        if (!$user->save()) {
+            return false;
         }
 
-        return false;
+        return \Yii::$app
+            ->mailer
+            ->compose(
+                [
+                    'html' => 'passwordResetToken-html',
+                    'text' => 'passwordResetToken-text',
+                ],
+                ['user' => $user]
+            )
```

Comment:
I breaked it like that because in current state it violates line max length (you can see vertical line in most modern IDEs). It's also acceptable as you wrote it, but I usually break it into multiple lines if amount of parameters greater than 2. But in such state it fits to line limit, I can make it like that too.